plugins/palo_alto_cortex_xdr/unit_test/test_get_alerts.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from unittest.mock import patch
+from icon_palo_alto_cortex_xdr.connection.connection import Connection
+from icon_palo_alto_cortex_xdr.triggers.get_alerts import GetAlerts
+import logging
+import json
+import timeout_decorator
+
+
+# This will catch timeout errors and return None. This tells the test framework our test passed.
+# This is needed because the run function in a trigger is an endless loop.
+def timeout_pass(func):
+    def func_wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except timeout_decorator.timeout_decorator.TimeoutError as e:
+            print(f"Test timed out as expected: {e}")
+            return None
+
+    return func_wrapper
+
+
+# This mocks komand.Trigger.send
+# We need this to fake out the plugin into thinking it's sending output in the trigger's run function
+class fakeSender:
+    def send(params):
+        print(params)
+
+
+# Test class
+class TestGetAlerts(TestCase):
+    @timeout_pass
+    @timeout_decorator.timeout(30)
+    @patch("insightconnect_plugin_runtime.Trigger.send", side_effect=fakeSender.send)
+    def test_integration_get_alerts(self, mockSend):
```

Comment:
This needs to be removed. The integration tests will fail the new unit test runner.

Reply:
Will do, what about the other tests such as `test_get_incidents.py` should those also be removed? as far as I can tell the get incidents test is more or less the same as the get_alerts tests.

Reply:
Yea, get rid of anything that "test_integartion..."